Tests/LibWeb/Layout/input/table-auto-height.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html><html><head><style>
+    body {
+        font-family: 'DejaVu Sans';
```

Comment:
I know it won't work and will be confusing for somebody trying to run layout tests without this font installed but probably we can start from tests that work at least on debian where this font supposed to be preinstalled :)